tests/storage/test_delete.py

```diff
@@ -0,0 +1,76 @@
+import hashlib
+import os
+import glob
+import traceback
+import re
+import binascii
+import logging
+import sys
+sys.path.append("..")
+import time
+from logger import Logger, LOG_FOLDER, LOG_EXTENSION
+from test_main import Test
+import argparse
+
+
+extension = ".txt"
+
+def get_args():
+    parser = argparse.ArgumentParser(description='Arguments', formatter_class = argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument( "--file_name", dest="file_name",
+                        required = True,
+                        help='Name of the file')
+    parser.add_argument("--dest", dest="path",
+                        required=True,
+                        help='Specify the path')
+    args = parser.parse_args()
+    return args
+
+
+class Delete(Test):
+    """
+
+    Checks if a file gets deleted or not
+    """
+
+    def __init__(self, file_name, dest_path):
+        self.ref_timestamp = int(time.time())
+        self.storage_path = dest_path
+        self.file_name = file_name
```

Comment:
You should not duplicate internal vars and the ones in `params`. Just use params.
Then, whenever you need the values just use that. Example: `os.remove(self.params['file_name'])`.